postgres.compose.yaml

```diff
@@ -0,0 +1,35 @@
+# Drupal with MySQL / MariaDB
+#
+# Access via "https://localhost"
+#
+# During initial Drupal setup,
+# Database type: PostgreSQL
+# Database name: postgres
+# Database username: postgres
+# Database password: example
+# ADVANCED OPTIONS; Database host: postgres
+
+services:
+
+  drupal:
```

Comment:
This should be possible to override a `db` service instead of copy/pasting the whole file.
Basically, name both the Postgres and the MariaDB service `db`, so it will be possible to run `docker compose -f compose.yaml -f compose.postgres.yaml up` to start the project with Postgres.

Alternatively, it should be possible to move the definition of the `drupal` service in a common file included by both the Maria and Postgres variants.

https://docs.docker.com/compose/multiple-compose-files/